### PR TITLE
REL: PyMC3 RC6

### PR DESCRIPTION
**Pull request by springcoil**
Updating to RC6.

### CODE REVIEW

**Comment by springcoil** [comment]
```+source activate /Users/travis/miniconda3/envs/_test
++[[ -n 3.2.51(1)-release ]]
++_SCRIPT_LOCATION=/Users/travis/miniconda3/envs/_test/bin/activate
++_SHELL=bash
+++dirname /Users/travis/miniconda3/envs/_test/bin/activate
++_CONDA_DIR=/Users/travis/miniconda3/envs/_test/bin
++'[' 1 -gt 1 ']'
++case "$(uname -s)" in
+++uname -s
++EXT=
++[[ -n 3.2.51(1)-release ]]
+++basename /Users/travis/miniconda3/conda-bld/test-tmp_dir/conda_test_runner.sh
++[[ conda_test_runner.sh == \a\c\t\i\v\a\t\e ]]
++'[' 1 -eq 0 ']'
++args=/Users/travis/miniconda3/envs/_test
++/Users/travis/miniconda3/envs/_test/bin/conda ..checkenv bash /Users/travis/miniconda3/envs/_test
++((  0 != 0  ))
++source /Users/travis/miniconda3/envs/_test/bin/deactivate
+++[[ -n 3.2.51(1)-release ]]
+++_SCRIPT_LOCATION=/Users/travis/miniconda3/envs/_test/bin/deactivate
+++_SHELL=bash
++++dirname /Users/travis/miniconda3/envs/_test/bin/deactivate
+++_CONDA_DIR=/Users/travis/miniconda3/envs/_test/bin
+++case "$(uname -s)" in
++++uname -s
+++EXT=
+++[[ 1 > 0 ]]
+++key=/Users/travis/miniconda3/envs/_test
+++case $key in
+++shift
+++[[ 0 > 0 ]]
+++[[ -n 3.2.51(1)-release ]]
++++basename /Users/travis/miniconda3/conda-bld/test-tmp_dir/conda_test_runner.sh
+++[[ conda_test_runner.sh == \d\e\a\c\t\i\v\a\t\e ]]
+++[[ -z /Users/travis/.rvm/gems/ruby-2.0.0-p481/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p481@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p481/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin ]]
+++((  0 == 0  ))
+++_CONDA_D=/Users/travis/miniconda3/etc/conda/deactivate.d
+++[[ -d /Users/travis/miniconda3/etc/conda/deactivate.d ]]
+++unset CONDA_DEFAULT_ENV
+++unset CONDA_PREFIX
+++export PATH=/Users/travis/.rvm/gems/ruby-2.0.0-p481/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p481@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p481/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
+++PATH=/Users/travis/.rvm/gems/ruby-2.0.0-p481/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p481@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p481/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
+++unset CONDA_PATH_BACKUP
+++export PS1=
+++PS1=
+++unset CONDA_PS1_BACKUP
+++[[ -n 3.2.51(1)-release ]]
+++hash -r
+++/Users/travis/miniconda3/envs/_test/bin/conda ..activate bash /Users/travis/miniconda3/envs/_test
++_NEW_PART=/Users/travis/miniconda3/envs/_test/bin
++((  0 == 0  ))
++export CONDA_PATH_BACKUP=/Users/travis/.rvm/gems/ruby-2.0.0-p481/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p481@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p481/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
++CONDA_PATH_BACKUP=/Users/travis/.rvm/gems/ruby-2.0.0-p481/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p481@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p481/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
++export CONDA_PS1_BACKUP=
++CONDA_PS1_BACKUP=
++export PATH=/Users/travis/miniconda3/envs/_test/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p481/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p481@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p481/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
++PATH=/Users/travis/miniconda3/envs/_test/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p481/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p481@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p481/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
++[[ '' == */* ]]
++export CONDA_DEFAULT_ENV=/Users/travis/miniconda3/envs/_test
++CONDA_DEFAULT_ENV=/Users/travis/miniconda3/envs/_test
++firstpath=/Users/travis/miniconda3/envs/_test/bin
+++echo /Users/travis/miniconda3/envs/_test/bin
+++sed 's|/bin$||'
++export CONDA_PREFIX=/Users/travis/miniconda3/envs/_test
+++/Users/travis/miniconda3/envs/_test/bin/conda ..changeps1
++'[' 1 = 1 ']'
+++grep -q CONDA_DEFAULT_ENV
++export 'PS1=(/Users/travis/miniconda3/envs/_test) '
++PS1='(/Users/travis/miniconda3/envs/_test) '
++_CONDA_D=/Users/travis/miniconda3/envs/_test/etc/conda/activate.d
++[[ -d /Users/travis/miniconda3/envs/_test/etc/conda/activate.d ]]
++unset CONDA_PATH
++[[ -n 3.2.51(1)-release ]]
++hash -r
+/Users/travis/miniconda3/envs/_test/bin/python -s /Users/travis/miniconda3/conda-bld/test-tmp_dir/run_test.py
===== testing package: pymc3-3.0rc6-py27_0 =====
import: 'pymc3'
/Users/travis/miniconda3/envs/_test/lib/python2.7/site-packages/matplotlib/font_manager.py:280: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
  'Matplotlib is building the font cache using fc-list. '
Traceback (most recent call last):
  File "/Users/travis/miniconda3/conda-bld/test-tmp_dir/run_test.py", line 26, in <module>
    import pymc3
  File "/Users/travis/miniconda3/envs/_test/lib/python2.7/site-packages/pymc3/__init__.py", line 21, in <module>
    from .plots import *
  File "/Users/travis/miniconda3/envs/_test/lib/python2.7/site-packages/pymc3/plots.py", line 3, in <module>
    import matplotlib.pyplot as plt
  File "/Users/travis/miniconda3/envs/_test/lib/python2.7/site-packages/matplotlib/pyplot.py", line 115, in <module>
    _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
  File "/Users/travis/miniconda3/envs/_test/lib/python2.7/site-packages/matplotlib/backends/__init__.py", line 32, in pylab_setup
    globals(),locals(),[backend_name],0)
  File "/Users/travis/miniconda3/envs/_test/lib/python2.7/site-packages/matplotlib/backends/backend_macosx.py", line 19, in <module>
    from matplotlib.backends import _macosx
RuntimeError: Python is not installed as a framework. The Mac OS X backend will not be able to function correctly if Python is not installed as a framework. See the Python documentation for more information on installing Python as a framework on Mac OS X. Please either reinstall Python as a framework, or try one of the other backends. If you are using (Ana)Conda please install python.app and replace the use of 'python' with 'pythonw'. See 'Working with Matplotlib on OSX' in the Matplotlib FAQ for more information.
TESTS FAILED: pymc3-3.0rc6-py27_0
``` 
Anyone have any suggestions on how to fix this?

**Comment by springcoil** [comment]
The error looks related to https://github.com/conda-forge/matplotlib-feedstock/pull/92 cc @ocefpaf  can you help?

**Comment by dopplershift** [comment]
The problem is (for the last failed build) that it picked up the _1 build rather than the _2 build, which has the fix. I have no idea why, though.

**Comment by springcoil** [comment]
Do you have any suggestions on how to change the build number?

**Comment by dopplershift** [comment]
No idea--conda should be pulling down the latest build since there were no changes to any dependencies. It works for me locally.

@msarahan any idea why conda wouldn't be grabbing the latest build number?

**Comment by dopplershift** [comment]
Ah, I see now--somehow the 2.0rc2 pacakges ended up on the main channel. I've asked to have the packages re-labelled, at which point you should be fine with the 1.5.3 packages.

**Comment by ocefpaf** [comment]
> Ah, I see now--somehow the 2.0rc2 pacakges ended up on the main channel. I've asked to have the packages re-labelled, at which point you should be fine with the 1.5.3 packages.

Should be fixed now. We have a bug that when we merge something that is supposed to be sent to `main` everything gets the `main` label. Until we fix that we need to manually remove the label from the `rc` packages.

**Comment by springcoil** [comment]
Ok just removing the python 3.6 to get this to work.

**Comment by springcoil** [comment]
I can't seem to remove the Python 3.6 - can you help @ocefpaf @msarahan ? This release is only tested on python 3.5 and 2.7

**Comment by ocefpaf** [comment]
Because all the dependencies are available `conda-smithy` will add Python 3.6 automatically. So you need to skip it in the build section. Something like:

```yaml
skip: True  # [py36]
```

I am not sure if the pre-processor selector `py36` already exists, so maybe you will need to use `py>35` instead.

**Comment by springcoil** [comment]
Is that in the `recipe/meta.yaml`

**Comment by twiecki** [comment]
Why does 3.6 not work?

**Comment by springcoil** [comment]
It says it conflicts with python 3.6 and the new release version. Should it
work? c.c @fonnesbeck

**Comment by twiecki** [comment]
What is it?

**Comment by springcoil** [comment]
Here is the error message
```
...Error: The following specifications were found to be in conflict:
  - pymc3 3.0rc6 py36_0
  - python 3.6*
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by msarahan** [comment]
@springcoil that's just conda's solver providing difficult hints.  It's supposed to be improved in conda 4.3.

Basically, what this means is that some dependency is not built for Python 3.6, so your package here  is currently not installable for python 3.6.

**Comment by twiecki** [comment]
I bet that's theano. They just made it work with python 3.6 on windows. Can we just disable the build for windows python 3.6?

**Comment by springcoil** [comment]
That's logical. I'm trying to disable this for windows python 3.6 - that's what my last commit is.

**Comment by springcoil** [comment]
And 3.6 on Mac also doesn't work which might be theano too. I propose ignoring python 3.6 in total for this release candidate.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [py>35]
```
Comment (springcoil) [inline recipe/meta.yaml]: Is there anything wrong with this @msarahan

**Comment by springcoil** [comment]
I seem unable to remove python 3.6 from this... help :) cc @ocefpaf @msarahan

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipe/meta.yaml]: Looks good to me. We can also work on getting the dependencies compiled for `Python 3.6` and make it work.

**Comment by springcoil** [comment]
Yeah might be a good idea to get theano sorted

**Comment by springcoil** [comment]
Ok looks ready to merge :)

**Comment by ericmjl** [comment]
@springcoil merging!

**Pull request merged**